src/components/input/index.js

```diff
@@ -1 +1 @@
-module.exports = require('./code/index');
+module.exports = require('./input.jsx');
```

Comment:
they're needed for components that are included by other components at the moment so that the tests work on the jsx, and the dist code works too... for example
`input = require('../input')`
works in the dist folder, and also in the src folder if we have this. Without this we have
`input = require('../input.jsx')`
which works in the src folder but then breaks in the dist folder.
There are other ways to do this, we could have 
```// input.jsx
module.exports = require('./input')
```
in the dist folders too though that also seems a bit odd... open to suggestions here. I guess having a `module.exports = require('./foo.jsx');` in every folder is best, if we need to keep folders..